charts/rd-user-profile-api/values.preview.template.yaml

```diff
@@ -9,11 +9,10 @@ java:
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
 
```

Comment:
remove aadIdentityName from this file and increase chart version by 1 and then see PR is green

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/252/commits/346362c6450c88462f59bf41431bfe83d48238ac